data/categories.txt

```diff
@@ -15717,6 +15717,9 @@ zh-Hant:消防栓
 fa:شیر آتش‌نشانی
 mr:अग्निशामक नाळ|फायर हायड्रंट
 
+emergency-lifeguard
+en:Lifeguard|lifesaver
```

Comment:
"lifesaver" might be confused with the rubber rings for emergencies - but actually i don't think we display them yet (could be a good idea)

Reply:
https://wiki.openstreetmap.org/wiki/Tag%3Aemergency%3Dlifeguard says people use "lifesaver" also..